src/components/offer-card/offer-avatar-and-rating/OfferAvatarAndRating.jsx

```diff
@@ -6,14 +6,19 @@ import AppRatingMobile from '~/components/app-rating-mobile/AppRatingMobile'
 
 import { styles } from '~/components/offer-card/offer-avatar-and-rating/OfferAvatarAndRating.styles'
 
+
 const OfferAvatarAndRating = ({ imgSrc, rating, totalReviews }) => {
   return (
     <Box sx={styles.container}>
       <Avatar src={imgSrc} sx={styles.avatar} />
       <AppRating readOnly showNumber sx={styles.rating} value={rating} />
-      <AppRatingMobile reviewsCount={totalReviews} showStar={false} />
+      <Box sx={styles.reviewsCountContainer}>
```

Comment:
box over box over box. don`t do so, align items in container left and avatar alignSelf: center